tests/scripts/run_tests.sh

```diff
@@ -4,7 +4,7 @@ set -ex
 
 # delete any __pycache__ folders to avoid hard-to-debug caching issues
 find . -name __pycache__ -type d -exec rm -r {} +
-py.test -v ${PYTEST_ARGS} "${PYTEST_JUNIT}" tests
+/home/user/.local/bin/py.test -v ${PYTEST_ARGS} "${PYTEST_JUNIT}" tests
```

Comment:
I assume this particular script is not executed locally at all but always within the docker container context?

If so, what do you think to use `$HOME` rather than `/home/user`?

Reply:
yes, I will change to `$HOME` everywhere instead of `/home/user`

Reply:
As `$HOME/.local/bin` is in the `PATH`, is the absolute path needed here? Removing it would make this script usable from the outside too, given the tools are in place.

Reply:
`$HOME/.local/bin` is not in the `PATH`

Reply:
let's add it to the `PATH`